sky/clouds/aws.py

```diff
@@ -10,6 +10,9 @@
 import typing
 from typing import Dict, Iterator, List, Optional, Tuple, Any
 
+import boto3
+import botocore
```

Comment:
Don't import it globally, as it will cause error when the user don't have this packages installed. Check the comment below.